core/src/main/java/dev/hypera/scaffolding/region/Region.java

```diff
@@ -187,10 +185,10 @@ public int hashCode() {
     @Override
     public boolean equals(Object obj) {
         return obj == this || (
-            obj instanceof Region region
-                && Objects.equals(this.instance, region.getInstance())
-                && Objects.equals(this.lower, region.getLower())
-                && Objects.equals(this.upper, region.getUpper())
+                obj instanceof Region region
+                        && Objects.equals(this.instance, region.getInstance())
+                        && Objects.equals(this.lower, region.getLower())
+                        && Objects.equals(this.upper, region.getUpper())
```

Comment:
These four lines shouldn't be indented any more than they were.

Reply:
Readable & Maintainable code is important for myself.

Reply:
If you really wish, i could reverse it.